packages/langium-cli/CHANGELOG.md

```diff
@@ -1,6 +1,37 @@
 # Change Log of `langium-cli`
 
-## v1.2.1 (June. 2023)
+## v2.0.0 (Jul. 2023)
+
+### Railroad Syntax Diagrams
+
+With the introduction of [`langium-railroad`](https://github.com/langium/langium/tree/main/packages/langium-railroad), the CLI is now capable of generating railroad syntax diagrams for your language ([#1075](https://github.com/langium/langium/pull/1075)).
+To generate them to a file, use the following example config:
+
+```json
+{
+    ...
+    "languages": [{
+        ...
+        "railroad": {
+            "out": "docs/syntax-diagram.html"
+        }
+    }],
+    ...
+}
+```
+
+### Generated Terminal Definitions
+
+The generated `ast.ts` file will now also contain an object containing all regular expressions used by your grammar's terminal rules ([#1097](https://github.com/langium/langium/pull/1097)).
+This allows to more easily reuse those regular expressions in your code.
+
+### General Improvements
+
+* The CLI can now resolve grammar imports transitively ([#1113](https://github.com/langium/langium/pull/1113)).
```

Comment:
I think that PR affects only the language server, not the CLI, right? So its mention should be moved to `langium-vscode`.

Reply:
The CLI also wasn't capable of handling transitive imports, so I'll add it to both the CLI and the vscode extension.